infra/hack/kubernetes/strims/30-svc.yaml

```diff
@@ -79,9 +79,8 @@ spec:
       - name: strims
         image: ghcr.io/memelabs/strims/svc:latest
         imagePullPolicy: Always
-        command: ["sh", "-c"]
         args:
-        - /app/svc run -config /etc/strims/config.yaml -host-ip "$(cat /node-labels/public-ip)";
+        - run -config /etc/strims/config.yaml -host-ip "$(cat /node-labels/public-ip)";
```

Comment:
I think this is required for the `"$(cat /node-labels/public-ip)";` hack

Reply:
Ah, I'll need to figure out where ko puts the binary.